FORMAT.md

```diff
@@ -135,6 +136,32 @@ The joint refers to a massless degree of freedom.
 <joint axis="rx"/>
 ```
 
+### `<end-effector>`
+
+An end effector refers to a component at the end of a kinematic chain (e.g., that has no children).  The key aspect of the end effector is that it identifies the location of an "end effector frame" at a specified relative position.
+
+(Note that HRDFs of version <= 1.1.0 do not explicitly have the notion of an end effector frame, so when being loaded into a compliant parser, the API adds an implicit "end effector frame" to the end of the chain of elements).
+
+By default, the end effector is massless with an identity transformation to the end effector frame from its input.
+
+**Required attributes:**
+
+(none)
+
+**Optional attributes:**
+- `mass` (floating point formula, kg) Defaults to 0.
+- `com_rot` (rotation matrix) the orientation of the center of mass (used for simplifying the inertia tensor description if desired).  Defaults to identity.
+- `com_trans` (translation vector, m) The position of the center of mass.  Defaults to (0,0,0).
+- `end_effector_rot` (rotation matrix): the orientation of the end effector frame relative to the end effector's input.  Defaults to identity.
+- `end_effector_trans` (translation vector, m): The position the end effector frame relative to the end effector's input.  Defaults to (0,0,0).
```

Comment:
Naming here -- I'm open to suggestions. `end_frame_trans`? `end_effector_trans`? `frame_trans`? `end_trans`?  Nothing seems quite right to me...

Reply:
Why do we need an `end_effector_trans` inside an `<end_effector>` element at all? Why not just stick with `output_trans` / `output_rot` and stay consistent with the `<rigid_body>` we already have?

Reply:
An `<end-effector>` can probably be identical to a `<rigid-body>` with some special behavior in the API. It'd make parsing easier if both had the same fields.

Reply:
The reason I was thinking different fields is because of the API terminology:
"GetFrames" can take COM, Output, or EndEffector frames. So to me, calling the frame here an "output" makes it really confusing as to why an EndEffector has no output in the API

Reply:
Why would the end-effector not have an output in the API? `Output` returns all output frames, and I'd assume that includes the end-effector body. The get `EndEffectorFrame` is just a convenience wrapper that lets you get the last frame without having to throw away the others.